scalableminds/webknossos
# Added yes/no messages for annotation deletion

See [discuss for problem description](https://discuss.webknossos.org/t/cancel-a-task/468)

### Mailable description of changes:
- [Dashboard.Task] Task deletion modal buttons should now read as "yes" and "no"

### Steps to test:
- Create Task & request it
- In Dashboard.Tasks click on "Cancel Task" --> Modal buttons should be "yes" / "no"
- In Admin.Task List find the task. Expand the sub table to see all assigned instances. Click on "Cancel Task" --> Modal buttons should be "yes" / "no"

### Issues:
- fixes https://discuss.webknossos.org/t/cancel-a-task/468

------
